### ISSUE TITLE
Image url show internal pasteur URL instead of connect's

### ISSUE THREAD

**Issue by r03ert0** [ConnectProject/mavoix-app#47]
The images uploaded by the user have the internal pasteur url (https://ghfc17.hosting.pasteur.fr/...). This should be replaced by the ma-voix url: https://mavoix.connect-project.io/...

**Comment by ntraut** [comment]
It's because I made the config at the time when we did not have https://mavoix.connect-project.io/. Now it should be fixed.

**Comment by ntraut** [comment]
The Parse objects `TabItem` and `Asset` still contain the ghfc17 url since they are stored with it when they are created. The url is updated by the app when it receives the object so the request is now made to the mavoix url. I think we could avoid that translation but we would have to change the way `TabItem` and `Asset` objects are stored.

**Comment by r03ert0** [comment]
Yes. Better do these basic structural changes now, that we don't have any real users. If needed, we can just erase the whole DB.
